chromadb/test/property/strategies.py

```diff
@@ -408,6 +412,8 @@ def document(draw: st.DrawFn, collection: Collection) -> types.Document:
     # For cluster tests, we want to avoid generating documents of length < 3.
     # We also don't want them to contain certan special
     # characters like _ and % that implicitly involve searching for a regex in sqlite.
+
+    blacklist_categories: Tuple[str, ...] = ()
```

Comment:
nit: why define this as an empty tuple and redefine it below, we can just define it correctly the first time

Reply:
fixed